vendor/engines/bulk_email/spec/services/bulk_email/recipient_searcher_spec.rb

```diff
@@ -73,6 +73,33 @@ def place_order(purchaser:, product:, account:)
     end
   end
 
+  describe "#search_params_as_hidden_fields" do
+    let(:params) do
+      {
+        bulk_email: { user_types: [:customers, :account_users] },
+        commit: "Submit",
+        facility_id: facility.id,
+        start_date: "1/1/2009",
+        end_date: "12/31/2016",
+        products: [1, 3, 5],
+      }
+    end
+    let(:fragment) { Nokogiri::HTML(searcher.search_params_as_hidden_fields.join("")) }
```

Comment:
I just found `Capybara.string`, which in theory _should_ work here to give you Capybara's matchers. For some reason, though, I couldn't make it work. Didn't seem to parse the string correctly.